sprints/2024/02_08/ztf_bts-ngc.ipynb

```diff
@@ -3,9 +3,7 @@
   {
```

Comment:
<p>Could you please sort imports here:</p><pre class="ql-syntax" spellcheck="false">import lsdb
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
from astropy.coordinates import SkyCoord
from astropy.table import Table
from astroquery.vizier import Vizier
from dask.distributed import Client
</pre><p><code></code>`</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/notebooks_lf/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>